### PR TITLE
small bench changes

### PR DESCRIPTION
**Pull request by naomijub**

### CODE REVIEW

File: src/edn.rs [hunk 1]
```diff
@@ -1,11 +1,28 @@
 pub fn field_to_keyword(field_name: &str) -> String {
-    let mut keyword = field_name
-        .to_lowercase()
+    let mut keyword = String::from(':');
```
Comment (otaviopace) [inline src/edn.rs:2]: Nice!

File: Cargo.toml [hunk 2]
```diff
@@ -18,12 +18,17 @@ proc-macro = true
...
 trybuild = { version = "1.0", features = ["diff"] }
+criterion = "0.3"
 
 [dev-dependencies.cargo-husky]
 version = "1"
 default-features = false
 features = ["user-hooks"]
+
+[[bench]]
+name = "deser"
```
Comment (otaviopace) [inline Cargo.toml:33]: I don't see why the bench can't be `deserialize`. What do you think?

Comment (otaviopace) [reply Cargo.toml:33]: Since the test and implementation are the full word.

Comment (naomijub) [reply Cargo.toml:33]: just lazyness

**Pull request merged**